tests/Behat/AuthenticationContext.php

```diff
@@ -143,4 +165,28 @@ public function allowExternalUsers(): void {
     $this->configContext->setConfig('oe_authentication.settings', 'assurance_level', 'LOW');
   }
 
+  /**
+   * Configures oe_authentication module to allow to use user delete options.
+   *
+   * @Given the site is configured to display user delete options
```

Comment:
We _could_ call this "is configured to not restrict user delete options".
But to me it is not a blocker.

Reply:
so it would be
```suggestion
   * @Given the site is configured to not restrict user delete options
```